mega-service-agentQnA/launch_agents.sh

```
@@ -0,0 +1,57 @@
+echo "Launching worker agents..."
+cd $WORKDIR/GenAIComps/examples/AgentQnA
+bash launch_workers.sh &
+WORKERS_PID=$!
+echo "Worker agents started (PID: $WORKERS_PID)"
+
+# Wait briefly for workers to initialize
+sleep 5
+
+# Start the main application
+echo "Starting AgentQnA application..."
+cd $WORKDIR/GenAIComps
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)